tests/spread/integration/perl/task.yaml

```
@@ -0,0 +1,6 @@
+summary: Integration tests for perl
+
+execute: |
+  rootfs=$(install-slices base-files_base bash_bins coreutils_bins perl_bins)
+
+  chroot "$rootfs" perl -e 'print "hello world\n";' | grep "hello world"
```

The spread tests are failing because /dev/null is not found in the chrooted system, so please mount /dev.

```suggestion
  mount --bind /dev "$rootfs/dev"
  chroot "$rootfs" perl -e 'print "hello world\n";' | grep "hello world"
```

---

`mount` is not available in a non-privileged container.